modules/install/src/test/scala/coursier/install/InstallTests.scala

```diff
@@ -791,6 +797,13 @@ object InstallTests extends TestSuite {
         val launcher = installDir0.actualDest(id)
         assert(Files.isRegularFile(launcher))
 
+        // for a prebuilt launcher, we expect the path of the launcher to appear somewhere in the script
+        val scala3path = {
+          val original = "github.com/scala/scala3/releases/download/3.3.3/scala3-3.3.3.zip/scala3-3.3.3/bin/scala"
+          if (currentOs == "windows") original.replace('/', '\\')
```

Comment:
I use `currentOs` here because if ran from e.g. macOS then the Mac-formatted path will actually be in the launcher.